tests/test_interpolate.py

```diff
@@ -339,27 +375,27 @@ def test_duplicate_idxs_period_pairs():
     ff = Forcefield.load_default()
     st = SingleTopology(mol_a, mol_b, core, ff)
 
-    duplicate_torsion_idxs = (0, 1, 3, 4)
+    duplicate_proper_idxs = (0, 1, 3, 4)
 
     counts_kv_src = dict()
-    for idxs, p in zip(st.src_system.torsion.potential.idxs, st.src_system.torsion.params):
+    for idxs, p in zip(st.src_system.proper.potential.idxs, st.src_system.proper.params):
         key = tuple(idxs)
         if p[2] == 2.0:
```

Comment:
Guessing you mean this comment to apply to improper torsions, since proper torsions already can have different periods:

```
  'ProperTorsion': { 'patterns': [ ['[*:1]-[#6X4:2]-[#6X4:3]-[*:4]', [[0.4834078610240992, 0.0, 3.0]]],
                                   ['[#6X4:1]-[#6X4:2]-[#6X4:3]-[#6X4:4]', [[0.3859268900186583, 0.0, 3.0], [1.3221171897434616, 3.141592653589793, 2.0], [1.358878183988969, 3.141592653589793, 1.0]]],
                                   ['[#1:1]-[#6X4:2]-[#6X4:3]-[#1:4]', [[0.7999501384731329, 0.0, 3.0]]],
```

Reply:
(oh sorry, I think you meant for this *molecule* in particular)... will fix.

Reply:
hm I tried the above but I'm seeing p[2]'s with values  of 4.0 as well. Not sure what's going on here, will revisit later.

Reply:
Ahh, sorry, mistaken comment

>Guessing you mean this comment to apply to improper torsions, since proper torsions already can have different periods:

Yes, I had misread this as applying to improper torsions (where [all periodicities are currently 2.0](https://github.com/proteneer/timemachine/blob/205b1f04ef7ce77614dcb5a00154fc6da80898cd/timemachine/ff/params/smirnoff_2_0_0_ccc.py#L411-L417)), but these are proper torsions.

>(oh sorry, I think you meant for this molecule in particular)

Sorry for ambiguity -- wasn't thinking of a particular test molecule

> Not sure what's going on here, will revisit later.

Non-issue -- mistaken comment